includes/free-trial/class-wc-calypso-bridge-free-trial-orders-changes.php

```diff
@@ -0,0 +1,60 @@
+<?php
+/**
+ * Contains the logic for override WooCommerce > orders screen
+ *
```

Comment:
Also, it'll be good to have the version version headers here similar to https://github.com/Automattic/wc-calypso-bridge/blob/e5b3f33a8d03b95e4c50dfafdb9c073721db00bd/class-wc-calypso-bridge-dotcom-features.php#L4-L8

Reply:
Do you know why we want the version headers in PHP files? 

It seems that we don't have version headers in Core or WCCOM and I don't think we always remember to bump the version when we change a PHP file. Besides, wc-calypso-bridge is likely not to be used by the community.

Reply:
Added in 36f0d6c135ad785d65b68e48ae7136210dab7051.

Reply:
Hmm, good question. When I suggested I thought of following a precedent. In my personal experience, I've found the `@since` version to be helpful when debugging, for example, to identify which versions a bug may have lived through. I don't know about the usefulness of `@version` though.

Not sure if there are more use cases, but I guess it wouldn't hurt to add them for now.